### ISSUE TITLE
Unable to open URL: https://api.twitch.tv/kraken/users.json

### ISSUE THREAD

**Issue by infiniteflow24** [streamlink/streamlink-twitch-gui#721]
## Bug report

- [x] I understand the difference between *Streamlink* and *Streamlink Twitch GUI*.
- [x] This is a bug report and I have read the contribution guidelines.

### Environment details

**Operating system and version:**  
win10 pro 1909 / 18363.815

**Streamlink Twitch GUI version:**  
1.9.1

**Streamlink version:**  
1.4.1

**Configuration details:**  

no Specific settings or other operating system details 

### Description

When I ran the gui about 4 days ago, it didn't work. (All the broadcasts on Twitch are same.) 
Like the attached screenshot.
need help. 

ex)
Found matching plugin twitch for URL twitch.tv/hanryang1125
Unable to open URL: https://api.twitch.tv/kraken/users.json (invalid literal for int() with base 10: b'\x00\x00\x00\x00')
Waiting for streams, retrying every 1.0 second(s)
Unable to open URL: https://api.twitch.tv/kraken/users.json (invalid literal for int() with base 10: b'\x00\x00\x00\x00')
Unable to open URL: https://api.twitch.tv/kraken/users.json (invalid literal for int() with base 10: b'\x00\x00\x00\x00')

### Reproduction steps

1. ...
2. ...
3. ...

### Log output

[2020-05-10T00:07:36.092Z][error][Application]
uncaughtException: Error: Buffer size limit exceeded

[2020-05-10T06:17:08.157Z][error][StreamingService]
Error: Process exited with code 1

```


### Additional comments, screenshots, etc.

![1234512345](https://user-images.githubusercontent.com/39779586/81508714-b3aef880-9340-11ea-9e5b-ab248e0a88e1.png)

**Comment by bastimeyer** [comment]
What you're seeing is the output of **Streamlink**. This error message has nothing to do with **Streamlink Twitch GUI**, as it's just a launcher for Streamlink.

All I can do is ask you whether you're sure you're using the latest Streamlink version, that it's properly configured in the Twitch GUI (see the streaming settings menu) and whether Streamlink itself is working fine.

Run Streamlink from the command line shell and verify that it's working correctly. If it's not, then open an issue on the Streamlink issue tracker.

From what it looks like though is that you're having some kind of encoding issue while reading the the API response from the first API call Streamlink has to make while trying to get the access token of the stream.

**Comment by infiniteflow24** [comment]
1. I'm sorry to ask you an unrelated problem.
2. Yes, i'm very sure that i'm using the latest version of Streamlink. (1.4.1)
3. I don't understand your answer, but thank you.

**Comment by bastimeyer** [comment]
> 3\. I don't understand your answer

You are trying to debug an issue on your system related to Streamlink, not Streamlink Twitch GUI. This means that you need to run Streamlink without the Twitch GUI.

See the Streamlink CLI documentation, which includes a brief tutorial.
https://streamlink.github.io/cli.html

Since you're a Windows user, press <kbd>Win+R</kbd> (which stands for run command) and enter `cmd`, which will open the (old) Windows command line shell in a terminal window. Then enter the following and check whether you're seeing the same error message:
```
streamlink --loglevel=debug twitch.tv/CHANNELNAME best
```

**Comment by infiniteflow24** [comment]
yes, now  i understand this issue is related to Streamlink. 

I use that command line -> same result.
(streamlink --loglevel=debug twitch.tv/CHANNELNAME best)


[cli][debug] OS:         Windows 10
[cli][debug] Python:     3.6.6
[cli][debug] Streamlink: 1.4.1
[cli][debug] Requests(2.23.0), Socks(1.7.1), Websocket(0.57.0)
[cli][info] Found matching plugin twitch for URL twitch.tv/hanryang1125
[plugin.twitch][debug] Getting live HLS streams for hanryang1125
error: Unable to open URL: https://api.twitch.tv/kraken/users.json (invalid literal for int() with base 10: b'\x00\x00\x00\x00')

(I've already deleted and reinstalled the Streamlink.)
thanks.

**Comment by bastimeyer** [comment]
Ok, please open a new issue on the Streamlink issue tracker with the full debug log. Thanks.
https://github.com/streamlink/streamlink/issues

**Comment by infiniteflow24** [comment]
ok. done.
thank you so much.
